packages/components/README.md

```diff
@@ -10,6 +10,7 @@ Install the module
 npm install @wordpress/components --save
 ```
 
+[Note: is the following paragraph still relevant?]
```

Comment:
We can probably remove this as IE is not supported in the editor anymore.

Reply:
We polyfill some functionalities in WP core even after we dropped support for IE 11. We need to reword it and link to how to use the polyfill shipped in `@wordpress/babel-preset-default`:

https://github.com/WordPress/gutenberg/tree/trunk/packages/babel-preset-default#polyfill

It's more about the most recent features that are part of the JavaScript language but some browser vendors still didn't implement them. See in WP core:

https://github.com/WordPress/WordPress/blob/master/wp-includes/js/dist/vendor/wp-polyfill.js

Reply:
I will look into this in a separate PR and report back here with updates.

Reply:
I will look into this in a separate PR and report back here with updates.

Reply:
Didn't need a separate PR. Updated the "polyfills" paragraph with references to the polyfill shipped in `@wordpress/babel-preset-default` in [`4409c6c` (#33960)](https://github.com/WordPress/gutenberg/pull/33960/commits/4409c6c9b394c04ae6b4774860212eaa7bfdec1e)